README.md

```diff
@@ -26,8 +26,50 @@ the right to make this assignment.
 
 -----
 
-Hasjob runs on [Python][] with the [Flask][] microframework. You will need
-to install all the requirements listed in `requirements.txt` using
+### Postgres
+
+Hasjob uses Postgres >=9.4 and Redis server for development. To set up a Postgres
+DB:
+
+On OS X:
+
+    $ createuser -d hasgeek
+    $ createdb -O hasgeek -W hasjob
+
+On any Linux distribution:
+
+    $ sudo -u postgres createuser -d hasgeek
+    $ sudo -u postgres createdb -O hasgeek hasjob
+
```

Comment:
Command taken from https://github.com/scrollback/scrollback/blob/master/tools/install.sh#L287
I have previously written a install script for Scrollback which configured Postgres on OSX + Linux systems. I have also tested this command on Ubuntu Trusty (via a machine on c9.io) and confirm that it works.